shoop/core/settings.py

```diff
@@ -69,12 +69,12 @@
 #:
 #: Each discount module may change the price of a product.  See
 #: `shoop.core.pricing.DiscountModule` for details.
-SHOOP_DISCOUNT_MODULES = []
+SHOOP_DISCOUNT_MODULES = ["catalog_campaigns"]
```

Comment:
I think we should. We are offering this app as a base discount / campaign implementation